### PR TITLE
Validate release artifacts post release

### PR DESCRIPTION
**Pull request by meain**
## Description

This takes care of validating all the release artifacts once we have completed the release step. Reworks https://github.com/alcionai/corso/pull/1360 as we are still deciding on Windows docker images.

## Type of change

- [ ] :sunflower: Feature
- [ ] :bug: Bugfix
- [ ] :world_map: Documentation
- [ ] :robot: Test
- [x] :computer: CI/Deployment
- [ ] :hamster: Trivial/Minor

## Issue(s)

* fixes https://github.com/alcionai/corso/issues/1328

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -496,3 +460,181 @@ jobs:
...
+        run: |
+          printf 'User-agent: *\nDisallow: /' > build/robots.txt
+
+      - name: Push website
+        run: |
+          aws s3 sync build "s3://${{ needs.SetEnv.outputs.website-bucket }}" --delete
+
+      - name: Invalidate cloudfront
+        run: |
+          aws cloudfront create-invalidation --distribution-id ${{ needs.SetEnv.outputs.website-cfid }} --paths "/*"
```
Comment (ryanfkeepers) [inline .github/workflows/ci.yml:590]: Since we're duplicating the behavior, this might be a candidate for a composite action.

**Comment by meain** [comment]
We should be removing "Publish-Website" as a required job. It is not required(Not from this change, but this changes need it to be remove to get unblocked).
/cc @vkamra

**Comment by vkamra** [comment]
> We should be removing "Publish-Website" as a required job. It is not required(Not from this change, but this changes need it to be remove to get unblocked). /cc @vkamra

Done

File: .github/actions/publish-website/action.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+name: Publish Website
```
Comment (vkamra) [inline .github/actions/publish-website/action.yml:1]: @meain - should we move these changes out of this PR?

Comment (vkamra) [reply .github/actions/publish-website/action.yml:1]: Ignore my comment - this is just refactoring.

**Review by vkamra — APPROVED** [review]

**Comment by meain** [comment]
@vkamra Not sure if it is GitHub tripping up, but the `Publish-Website` job still somehow seems to be shown as required in the GitHub UI.

**Comment by meain** [comment]
Removing `mergequeue` until we look into required jobs as it will bock others.

**Review by ryanfkeepers — APPROVED** [review]

**Pull request merged**